etc/picongpu/taurus-tud/V100_picongpu.profile.example

```diff
@@ -1,76 +1,83 @@
+## `picongpu.profile` for the `ml` partition on taurus@TU-Dresden
+## The `ml` partition is equipped with Nvidia V100 GPUs
+## Nodes in the `ml` partition are Power9 systems which is different
+## from the headnodes. Therefore, compile on a node!
+
 # Name and Path of this Script ############################### (DO NOT change!)
 export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
 
-# User Information ######################################### (edit those lines)
+# User Information ######################################### (edit these lines)
```

Comment:
please fix the typo in a follow-up PR for all other profile examples as well

Reply:
ping. do not change it here or change it in all please

Reply:
in preparation...